docs/CLI.md

```diff
@@ -189,6 +189,19 @@ Prevent tests from printing messages through the console.
 
 Alias: `-t`. Run only tests and test suites with a name that matches the regex.  For example, suppose you want to run only tests related to authorization which will have names like `"GET /api/posts with auth"`, then you can use `jest -t=auth`. 
 
+### `--testLocationInResults`
```

Comment:
I need to update https://facebook.github.io/jest/docs/en/configuration.html#testresultsprocessor-string as well